lib/core/seleniumRunner.js

```diff
@@ -563,8 +563,9 @@ class SeleniumRunner {
         // invocation. The script writer cannot do this in their
         // source because it would force the evaluation of the
         // function too soon and cause undefined reference errors.
-        script = '(' + func + ')();';
+        script = '( ' + func + ' )()';
         requires = category[scriptName].requires;
+        isAsyncOverride = category[scriptName].isAsync;
```

Comment:
I wondered about this.  Don't you need to remove an `isAsyncOverride = false` line or similar somewhere?

Reply:
I default to false when I `let` the variable at the beginning of the function. I `||` that with the `isAsync` parameter when calling `this.runScriptFromCategory`. I think that this is the right behavior?